cvat/apps/documentation/installation.md

```diff
@@ -373,45 +373,42 @@ This depends on the email server you are using and is not covered in this tutori
 [Django SMTP backend configuration](https://docs.djangoproject.com/en/3.1/topics/email/#django.core.mail.backends.smtp.EmailBackend)
 for details.
 
-### Serving over HTTPS
```

Comment:
Please update the table of contents too

Reply:
@azhavoro Sorry, I forget to update TOC. Will do it soon.